application/apps/indexer/parsers/src/dlt/fmt.rs

```diff
@@ -386,6 +402,19 @@ impl<'a> FormattableMessage<'a> {
                     payload_string,
                 ))
             }
+            PayloadContent::NetworkTrace(slices) => {
+                let payload_string = slices
+                    .iter()
+                    .map(|slice| format!("{:02X?}", slice))
+                    .collect::<Vec<String>>()
+                    .join("|");
+                Ok(PrintableMessage::new(
```

Comment:
Same as above if we want to use `reduce()`.

Reply:
I don't see how is reduce expressing the intent better here.